doc/source/examples/dem/sandpile-formation/sandpile-formation.rst

```diff
@@ -0,0 +1,305 @@
+      subsection wall 0
+        subsection point on wall
+          set x = 0
+          set y = 0
+        end
+        subsection normal vector
+          set nx = 0
+          set ny = 1
+        end
+        set start time = 0
+        set end time   = 4.5
+      end
+    end
+
+-----------------------
+Running the Simulation
+-----------------------
+
+The simulation can be launched with
+
+.. code-block:: text
+  :class: copy-button
+
+   mpirun -np 2 lethe-particles sandpile.prm
+
+
+
+---------------
+Post-processing
+---------------
+
+A Python post-processing code called ``sandpile-postprocessing.py`` is provided with this example. It is used to measure the height of the pile at each time set, starting at 10.02s so that the pile is already formed. It also calculates the angle of repose of the pile, based on the last frame.
+
+It compares the data generated by the simulation to data from J.Ai *et al.* [#Ai2010]_ for the selected rolling resistance model.
+
+It is possible to run the post-processing code with the following line. The arguments are the simulation path, the prm file name and the rolling resistance model used.
+
+.. code-block:: text
+  :class: copy-button
+
+    python3 sandpile-postprocessing.py  --folder ./ --prm sandpile.prm --rollingmethod epsd
+
+
+The code prints the values of R2 and the slope (from the regression) and the angle of repose.
+
+.. note::
+
+  The argument --rollingmethod can be either epsd, viscous or constant and should be the same as the "rolling resistance torque method" in the prm file.
+  The argument --regression can be added to plot the least squares regression used to calculate the angle of repose.
+
+.. important::
+
+    You need to ensure that ``lethe_pyvista_tools`` is working on your machine. Click `here <../../../tools/postprocessing/postprocessing.html>`_ for details.
+
+
+When you have launched the simulation and the post-processing (with the right argument) for each rolling resistance model (constant, epsd, viscous), launch the following to compare different models.
+
+.. code-block:: text
+  :class: copy-button
+
+    python3 sandpile-height-comparison.py
+
+
+-------
+Results
+-------
+
+Visualisation with Paraview
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation can be visualised using Paraview as seen below.
+
+.. figure:: images/formed-sandpile.png
+    :width: 400
+    :alt: Mesh
+    :align: center
+
+    Sandpile at the end of the simulation
+
+
+Evolution of the Height of the Pile
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following image compares the evolution of the height of the pile with the results of J. Ai *et al.*.
+
+.. image:: images/figure-height-comparison.png
+    :alt: Height comparison
+    :align: center
+
+
+
+
+The next image shows the evolution of the height of the pile with rolling resistance model epsd using different prn seeds.
+
+.. image:: images/figure-height-different-prn-seeds.png
+    :alt: Height comparison
+    :align: center
+
```

Comment:
I would maybe add a comment on what you observed here. Explore the results a bit more :)